### PR TITLE
Added preliminary readme

### PR DESCRIPTION
**Pull request by arpan-dhatt**
Completed preliminary README with instructions and references to other documentation. This will likely need changes and updates in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Setting up Remote Development
+Since setting up all the dependencies for a project is extremely challenging since they are platform development, you can test your code on the Raspberry Pi remotely.
+1. Get the Raspberry Pi IP address and password from the Discord server `#smart-mirror` channel. The IP might change, so be aware of that.
+2. In VSCode, make sure you've installed the "Remote Development - SSH" extension so you can remotely access the files on the Pi and run your code.
+3. [Follow the instructions](https://code.visualstudio.com/docs/remote/ssh#_connect-to-a-remote-host) for connecting to a remote host. **Note: the username is `pi`.**
+4. Go ahead and clone the repository to the Raspberry Pi using the [VSCode integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal) using the same commands you used in the **Contributing to the Repository** section. You might need to switch to your branch using `git checkout`.
```
Comment (sahiljain11) [inline README.md]: For the 'git checkout' part, add 'git checkout <branch_name> for specificity

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+1. First, you need to [fork this repository](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/about-forks) so you can make your own changes. At the top right of this page, there's a fork button. Press that, and you'll end up with a copy of this repo.
+2. Now that you have a fork, you're almost ready to start making changes. [Create a new branch](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository) in your repository, and name it whatever you'd like, such as the name of the feature you're about to start work on.
+3. Now you need to [clone your repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) to your computer and actually begin making changes.
```
Comment (sahiljain11) [inline README.md]: After this line, add a section on updating user.name and user.email. Make sure to specify for them NOT to add the --set-global flag

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# UT ECLAIR Smart Mirror Project
+This is the GitHub repository for ECLAIR's smart mirror project. Ultimately, this software will run on a Raspberry Pi and provide a useful interface so users can see the time, any todo items, the weather, etc. through a mirror.
+
+These instructions are not very thorough, so if you have any trouble and the linked details aren't sufficient, ask for clarification on the Discord server.
+
+## Getting Started
+1. Make sure you're on the ClickUp to look at what tasks are being worked on.
```
Comment (sahiljain11) [inline README.md]: Add a part for "Contact Sahil if you are not already on the clickup"

**Comment by arpan-dhatt** [comment]
Need to add
 * Setting Github Credentials Locally

File: README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+3. Now you need to [clone your repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) to your computer and actually begin making changes. 
+4. Switch to the branch you just created using `git checkout <branch-name>`. You can use VSCode or some other IDE to make changes. You might not have all the dependencies installed so there could be a lot of errors. You will be able to test your code on the Raspberry Pi using the sections below.
+5. After you're done making changes or at any frequency you'd like to "checkpoint" your code, run `git commit -am "<describe your changes>"` and `git push origin <branch-name>` to send the code to your forked repository.
+6. Once you're done with a feature or task and you'd like it to be reviewed, head back to GitHub and start a [Pull Request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork). You can see exactly what changes you made and describe them. Once you're done, you can finish by pressing "Create Pull Request".
```
Comment (sahiljain11) [inline README.md]: After this step, add a section describing the requirements for a pull request. This includes adding me for review, a image of things working, a proper title, bugs/problems you faced, what to watch out for, limitations on the module, etc.

**Review by sahiljain11 — CHANGES_REQUESTED** [review]
Overall mostly good.  Couple of smaller steps need to still be done, but good work homie!

**Review by sahiljain11 — APPROVED** [review]
LGTM!

**Pull request merged**